exercises/rest-api/RestApi.fs

```diff
@@ -1,3 +1,11 @@
 ﻿module RestApi
 
-// TODO: implement this module
\ No newline at end of file
+type RestApi(database : string) =
+
+    member this.Get (url:string) =
```

Comment:
You you remove the space after the method name and add a space before the type identifier?